src/getTime.js

```diff
@@ -18,7 +18,32 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  let strWithoutSumb = str.match(/[^,]/g).join('').split(' ');
```

Comment:
```suggestion
  let strWithoutSymbols = str.match(/[^,]/g).join('').split(' ');
```
Symbols
Не сокращай слова